### PR TITLE
Add OTA service

### PR DESCRIPTION
**Pull request by ditek**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1190,4 +1190,40 @@ in   | cmd.log.set_level     | string     |            |
...
+}
+{
+   "type": "evt.ota_end.report",
+   "serv": "ota",
+   "val_t": "str_map",
+   "val": {
+     "success": "true",
+      "error": ""
+   }
+}
```
Comment (bjhandeland) [inline README.md:1228]: Move this to a separate code block, i.e.

```json
{
   "type": "evt.ota_end.report",
   "serv": "ota",
   "val_t": "str_map",
   "val": {
     "success": "true",
      "error": ""
   }
}
```

File: README.md [hunk 2]
```diff
@@ -1190,4 +1190,40 @@ in   | cmd.log.set_level     | string     |            |
...
+#### Service name 
+
+`ota`
+
+#### Interfaces 
+
+Type | Interface               | Value type | Properties | Description 
+-----|-------------------------|------------|------------|--------------
+out  | evt.ota_progress.report | int_map    |            | Shows upgrade progress and estimated time left
+out  | evt.ota_end.report      | string_map |            | Sent on upgrade end with upgrade status
```
[the remark this answers is not in the document]
Comment (ditek) [reply README.md]: So only a period is needed, right?

**Comment by ditek** [comment]
Comments addressed :)

**Review by bjhandeland — APPROVED** [review]
Lgtm :)

**Pull request merged**